docs/quickstart/install-and-use-a-package-in-visual-studio-mac.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Install and use a NuGet package in Visual Studio for Mac
+description: A walkthrough tutorial on the process of installing and using a NuGet package in a Visual Studio for Mac project.
+author: jmatthiesen
+ms.author: jomatthi
+ms.date: 08/14/2019
+ms.topic: quickstart
+---
+
+# Quickstart: Install and use a package in Visual Studio for Mac
+
+NuGet packages contain reusable code that other developers make available to you for use in your projects. See [What is NuGet?](../What-is-NuGet.md) for background. Packages are installed into a Visual Studio for Mac project using the NuGet Package Manager. This article demonstrates the process using the popular [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json/) package and an .NET Core console project. The same process applies to any other Xamarin or .NET Core project.
+
+Once installed, refer to the package in code with `using <namespace>` where \<namespace\> is specific to the package you're using. Once the reference is made, you can call the package through its API.
+
+> [!Tip]
+> **Start with nuget.org**: Browsing *nuget.org* is how .NET developers typically find components they can reuse in their own applications. You can search *nuget.org* directly or find and install packages within Visual Studio as shown in this article. For general information, see [Find and evaluate NuGet packages](../consume-packages/finding-and-choosing-packages.md).
+
+## Prerequisites
+
+- Visual Studio 2019 for Mac.
+
+You can install the 2019 Community edition for free from [visualstudio.com](https://www.visualstudio.com/) or use the Professional or Enterprise editions.
+
+If you're using Visual Studio on Windows, see [Install and use a package in Visual Studio (Windows Only)](install-and-use-a-package-in-visual-studio).
+
+## Create a project
+
+NuGet packages can be installed into any .NET project, provided that the package supports the same target framework as the project.
+
+For this walkthrough, use a simple .NET Core Console app. Create a project in Visual Studio for Mac using **File > New Solution...**, select the **.NET Core > App > Console Application** template. Click **Next**. Accept the default values for **Target Framework** when prompted.
+
+Visual Studio creates the project, which opens in Solution Explorer.
+
+## Add the Newtonsoft.Json NuGet package
+
+To install the package, you use the NuGet Package Manager. When you install a package, NuGet records the dependency in  either your project file or a `packages.config` file (depending on the project format). For more information, see [Package consumption overview and workflow](../consume-packages/Overview-and-Workflow.md).
+
+### NuGet Package Manager
+
+1. In Solution Explorer, right-click **Dependencies** and choose **Add Packages...**.
+
+    ![Manage NuGet Packages command for project References](media/QS_Use_Mac-02-ManageNuGetPackages.png)
```

Comment:
The image appears too large. Padding with white on the right side (in the image) will make it look better.

Reply:
Fixed with my recent changes.